R/map.R

```diff
@@ -193,7 +194,7 @@ map_ <- function(.type,
 
   .f <- as_mapper(.f, ...)
 
-  if (isTRUE(.parallel)) {
+  if (is_crate(.f)) {
```

Comment:
Maybe move above `as_mapper` now?

Reply:
Yes, that should be moved up, done in [626fa80](https://github.com/tidyverse/purrr/pull/1185/commits/626fa801c7e8836bf310395910b21a743a6f7283).